model/log.go

```diff
@@ -262,6 +263,43 @@ func (ll *LogLine) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// GetBSON implements the bson.Getter interface.
+// When a LogLine is marshalled to BSON the driver will marshal the output
+// of this function instead of the struct.
+func (ll LogLine) GetBSON() (interface{}, error) {
+	return []interface{}{ll.Time, ll.Msg}, nil
+}
+
+// SetBSON implements the bson.Setter interface.
+// When a LogLine is unmarshalled from BSON the driver will call this function to
+// unmarshal into the LogLine.
+func (ll *LogLine) SetBSON(raw bson.Raw) error {
+	line := []interface{}{}
+	if err := raw.Unmarshal(&line); err != nil {
+		return &bson.TypeError{
+			Kind: raw.Kind,
+			Type: reflect.TypeOf(line),
+		}
+	}
+	if len(line) < 2 {
+		return errors.Errorf("line was of unexpected length %d", len(line))
+	}
+
+	time, ok := line[0].(time.Time)
+	if !ok {
+		return errors.Errorf("timestamp was of unexpected type '%T'", line[0])
```

Comment:
Nit: No need to put single quotes around %T since those are predefined values (we usually use single quotes when it is a string that may have special characters such as spaces that make it hard to delimit).

Reply:
done